source/tsconfig-json.d.ts

```diff
@@ -997,6 +1012,31 @@ declare namespace TsConfigJson {
 		@default false
 		*/
 		explainFiles?: boolean;
+
+		/**
+		Preserve unused imported values in the JavaScript output that would otherwise be removed.
+
+		Requires TypeScript version 4.7 or later.
+
+		@default true
+		*/
+		preserveValueImports?: boolean;
+
+		/**
+		List of file name suffixes to search when resolving a module.
+
+		Requires TypeScript version 4.5 or later.
+		*/
+		moduleSuffixes?: string[];
+
+		/**
+		Control what method is used to detect module-format JS files.
+
+		Requires TypeScript version 4.7 or later.
```

Comment:
`type-fest` requires TS 4.7, so this is moot.

Reply:
Because the only way to get this type is to already be on TS 4.7, so the message of requiring TS 4.7 is not needed.